Dockerfile.fips.standalone-infisical

```diff
@@ -69,13 +69,21 @@ RUN groupadd -r -g 1001 nodejs && useradd -r -u 1001 -g nodejs non-root-user
 
 WORKDIR /app
 
-# Required for pkcs11js
+# Required for pkcs11js and ODBC
 RUN apt-get update && apt-get install -y \
     python3 \
     make \
     g++ \
+    unixodbc \
```

Comment:
Hmmm so does the linux package also need to install these dependency?

Reply:
Yeah it's needed for the nodegyp install